src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,94 @@
+package core.basesyntax.service;
+
+import core.basesyntax.exception.InvalidUserDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_userIsNull_notOk() {
+        Exception exception = assertThrows(InvalidUserDataException.class, () -> {
+            registrationService.register(null);
+        });
+        assertEquals("User cannot be null", exception.getMessage());
+    }
```

Comment:
Great job testing the case when the user is null. This is an important edge case to consider.